app/views/AddBusinessTrade.scala.html

```diff
@@ -0,0 +1,122 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import auth.MtdItUser
+@import views.html.layouts.unifiedLayout
+@import views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import exceptions.MissingFieldException
+@import models.incomeSourceDetails.BusinessTradeForm
+@import helper._
+@import views.html.helpers.injected.ErrorSummary
+@import views.html.helpers.injected.InputHelper
+
+
+
+@this(
+mainTemplate: unifiedLayout,
+appConfig: config.FrontendAppConfig,
+h1: h1,
+h2: h2,
+p: p,
+link: link,
+form: FormWithCSRF,
+govukInsetText : GovukInsetText,
+govukButton : GovukButton,
+inputHelper: InputHelper,
+errorSummary: ErrorSummary
+)
+
+@(businessTradeForm: Form[BusinessTradeForm], postAction: Call, isUserAgent: Boolean)(implicit messages: Messages, user: auth.MtdItUser[_])
+
+@getMessage(key: String, args: String*) = @{
+messages(s"add-business-trade.$key", args: _*)
+}
+@field = @{
+businessTradeForm("trade")
+}
+@fieldError = @{
+field.error
+}
+
+
+@mainTemplate(pageTitle = messages(""),
+backUrl = if(!isUserAgent){Some("/report-quarterly/income-and-expenses/view/income-sources/add/business-start-date")}else{Some("/report-quarterly/income-and-expenses/view/agents/income-sources/add/business-start-date")},
+isAgent = isUserAgent,
+btaNavPartial = user.btaNavPartial) {
+
+
+@form(action = postAction) {
+@CSRF.formField
+
+@if(fieldError.isDefined){
+<div class="govuk-error-summary" data-module="govuk-error-summary">
```

Comment:
I would believe there is existing component we can use to show error summary => have a look how this done in EnterClientsUTR.scala.html => 

@errorSummary(
        form = clientUTRForm
    )